packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -232,6 +233,38 @@ describe("Garbage Collection Tests", () => {
 		);
 	});
 
+	it("Pending session expiry does not close container", () => {
+		let closeCalled = false;
+		const timenow = 5000; // arbitrary number
+		clock.tick(timenow);
+		gc = createGarbageCollector(
+			{ sessionExpiryTimerStarted: 0 },
+			undefined /* gcBlobsMap */,
+			undefined /* gcMetadata */,
+			() => {
+				closeCalled = true;
+			},
+		);
+		assert(closeCalled === false, "Close should have been called");
+	});
+
+	it("Pending session expiry closes container", () => {
+		let closeCalled = false;
+		const timethen = 12345;
+		const timenow = defaultSessionExpiryDurationMs;
+		clock.tick(timenow + timethen);
+		gc = createGarbageCollector(
+			{ sessionExpiryTimerStarted: timethen },
```

Comment:
I think the naming is a little off here - e.g. `timenow` isn't actually now (you're ticking something else).  How about this?

```suggestion
		const sessionExpiryTimerStarted = 12345;
		const timenow = sessionExpiryTimerStarted + defaultSessionExpiryDurationMs;
		clock.tick(timenow);
		gc = createGarbageCollector(
			{ sessionExpiryTimerStarted },
```